source/Calamari.Azure/Accounts/AccountFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Calamari.Deployment;
+using Octostache;
+
+namespace Calamari.Azure.Accounts
+{
+    public static class AccountFactory
+    {
+        public static Account Create(VariableDictionary variables)
```

Comment:
Wasn't clear on this.  Does this mean a variable set can contain only one variable of Account type?  Would that meant that the same deployment process can't access two azure accounts?  Did I miss something here?

Reply:
Yeah, that is how things currently work, you can only have 1 Account per variable set. Which equates to 1 account type per step in a process, because it comes down to the step running on a machine. There is some thinking going in to supporting multiple accounts, for things like connect to AWS and Azure at the same time, but this will be a big refactor across many things to make that work.